jbpm-event-emitters/jbpm-event-emitters-kafka/pom.xml

```diff
@@ -0,0 +1,45 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.jbpm</groupId>
+    <artifactId>jbpm-event-emitters</artifactId>
+    <version>7.45.0-SNAPSHOT</version>
```

Comment:
@fjtirado @elguardian you need to fix the version as on master is already 7.46.0-SNAPSHOT, this breaks the master

Reply:
fixed in https://github.com/kiegroup/jbpm/commit/c5796f6599ff55e48754b3d4e9d87445d506601a